test/unit/shipment_packer_test.rb

```diff
@@ -146,13 +146,37 @@ def test_cast_quantity_and_grams_to_int
     assert_equal 100, package.value
   end
 
-  def test_excess_packages
+  def test_excess_packages_raised_over_threshold
     assert_raises(ShipmentPacker::ExcessPackageQuantity) do
-      items = [{:grams => 1, :quantity => ShipmentPacker::EXCESS_PACKAGE_QUANTITY_THRESHOLD + 1, :price => 1.0}]
+      items = [{ grams: 1, quantity: ShipmentPacker::EXCESS_PACKAGE_QUANTITY_THRESHOLD + 1, price: 1.0 }]
       ShipmentPacker.pack(items, @dimensions, 1, 'USD')
     end
   end
 
+  def test_excess_packages_not_raised_at_threshold
+    assert_nothing_raised do
```

Comment:
You'll never need an `assert_nothing_raised`, that's what a test inherently does. Any test fails if something is raised.